src/core/include/mp-units/ostream.h

```diff
@@ -89,9 +89,12 @@ std::basic_ostream<CharT, Traits>& to_stream(std::basic_ostream<CharT, Traits>&
 
 MP_UNITS_EXPORT_BEGIN
 
+template<typename OStream, typename T>
+concept is_mp_units_stream = requires(OStream os, T v) { detail::to_stream_impl(os, v); };
```

Comment:
I wouldn't like to introduce new concepts for such things. Does the below work?

```suggestion
constexpr bool is_mp_units_stream = requires(OStream os, T v) { detail::to_stream_impl(os, v); };
```

Please also put it to a `detail` namespace and outside of the `MP_UNITS_EXPORT_BEGIN` `MP_UNITS_EXPORT_END` block.

Reply:
done